tests/integration_tests/performance/test_vsock_throughput.py

```diff
@@ -13,83 +13,89 @@
 from framework.artifacts import ArtifactCollection, ArtifactSet
 from framework.matrix import TestMatrix, TestContext
 from framework.builder import MicrovmBuilder
-from framework.statistics import core, consumer, producer, criteria,  types, \
-    function
+from framework.statistics import core, consumer, producer, criteria, types
 from framework.utils import CpuMap, CmdBuilder, run_cmd, eager_map, \
     get_cpu_percent
 from framework.utils_cpuid import get_cpu_model_name
 import host_tools.network as net_tools
-from integration_tests.performance.vsock_throughput_test_config import \
-    THROUGHPUT, DURATION, CPU_UTILIZATION_HOST, \
-    CPU_UTILIZATION_GUEST, CONFIG, CPU_UTILIZATION_VMM_TAG, \
-    CPU_UTILIZATION_VCPUS_TOTAL_TAG, THROUGHPUT_TOTAL, \
-    DURATION_TOTAL, IPERF3, BASE_PORT, \
-    IPERF3_CPU_UTILIZATION_PERCENT_OUT_TAG, IPERF3_END_RESULTS_TAG, \
-    TARGET_TAG, DELTA_PERCENTAGE_TAG, THROUGHPUT_UNIT, DURATION_UNIT, \
-    CPU_UTILIZATION_UNIT
+from integration_tests.performance.configs.vsock_throughput_test_config \
+    import THROUGHPUT, DURATION, CONFIG, CPU_UTILIZATION_VMM, \
+    THROUGHPUT_TOTAL, CPU_UTILIZATION_VCPUS_TOTAL, IPERF3_END_RESULTS_TAG, \
+    IPERF3, BASE_PORT, IPERF3_CPU_UTILIZATION_PERCENT_OUT_TAG, TARGET_TAG, \
+    DELTA_PERCENTAGE_TAG, THROUGHPUT_UNIT, DURATION_UNIT
 
 SERVER_STARTUP_TIME = CONFIG["server_startup_time"]
-
 VSOCK_UDS_PATH = "v.sock"
 
 
 def measurements_vsock():
     """Define the produced measurements for VSOCK workloads."""
     return [types.MeasurementDef(THROUGHPUT, THROUGHPUT_UNIT),
             types.MeasurementDef(DURATION, DURATION_UNIT),
-            types.MeasurementDef(CPU_UTILIZATION_HOST, CPU_UTILIZATION_UNIT),
-            types.MeasurementDef(CPU_UTILIZATION_GUEST, CPU_UTILIZATION_UNIT)]
+            types.MeasurementDef.cpu_utilization_vmm(),
+            types.MeasurementDef.cpu_utilization_vcpus_total()]
+
+
+def basic_vsock_stats():
```

Comment:
maybe we should rename the function. the stats are not basic, they just don't have pass criteria